architecture/core_concepts/routes.adoc

```diff
@@ -12,22 +12,28 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
-
-A defined route and the endpoints identified by its service can be consumed by a
-router to provide named connectivity that allows external clients to reach your
-applications. Each route consists of a route name (limited to 63 characters),
-service selector, and (optionally) security configuration.
+An OpenShift link:../architecture/core_concepts/routes.html[route] exposes a
+link:../architecture/core_concepts/pods_and_services.html#services[service] at a
+host name, like _www.example.com_, so that external clients can reach it by
+name.
+
+DNS resolution for a host name is handled separately from routing;
+your administrator may have configured a
+link:../../install_config/install/prerequisites.html#prereq-dns[DNS wildcard entry]
+that will always correctly resolve to the OpenShift router, or if using an unrelated
+host name you may need to modify its DNS records independently to resolve
+to the router.
```

Comment:
Took a wee bit more `parsing` power - can we break this up into 2/3 sentences ala: 
```
The DNS resolution for a route's host name is handled separately from routing.   
Your administrator may have ... that will resolve to the OpenShift router.  
If you are using a different host name, you may need to ...
```  
(parts snipped for brevity). 

There is also the last sentence that got chopped and it does have some info (maybe we can somehow keep that in):  
```  
Each route consists of a route name (limited to 63 characters), service selector,   
and (optionally) security configuration.  
```